HRIS.Services.Tests/Services/FieldCodeServiceUnitTests.cs

```diff
@@ -211,10 +174,10 @@ public async Task GetByCategoryPass(int categoryNumber)
 
         fieldCodes = new List<FieldCode>
         {
-            _fieldCodeDto3
+            _fieldCodeDto2
         };
```

Comment:
I would suggest cleaning this up because you are initializing `var fieldCodes` on line 169 but then on line 175 you overwrite the initial values.

So this
```
var fieldCodes = new List<FieldCode>
{
    _fieldCodeDto,
    _fieldCodeDto2
};

fieldCodes = new List<FieldCode>
{
    _fieldCodeDto2
};
```

should be 

```
var fieldCodes = new List<FieldCode>
{
    _fieldCodeDto2
};
```

Reply:
i see it, done